ktor-server/ktor-server-jetty-jakarta/build.gradle.kts

```diff
@@ -4,6 +4,7 @@ kotlin {
     sourceSets {
         jvmMain {
             dependencies {
+                api(kotlin("test"))
```

Comment:
could you tell me why do we need test dependency in the main module?

Reply:
Good catch, I must have got my build scripts mixed up.